src/generic/delete-modal/DeleteModal.jsx

```diff
@@ -5,17 +5,14 @@ import {
   Button,
   AlertModal,
 } from '@edx/paragon';
-import { useSelector } from 'react-redux';
 import { useIntl } from '@edx/frontend-platform/i18n';
 
-import { COURSE_BLOCK_NAMES } from '../constants';
-import { getCurrentItem } from '../data/selectors';
 import messages from './messages';
 
-const DeleteModal = ({ isOpen, close, onDeleteSubmit }) => {
+const DeleteModal = ({
```

Comment:
[nit]: 
- Please remove `import React from 'react';`
- Change the name of the folder and component to `ModalDelete`

Reply:
[FYI] @ruzniaievdm @monteri @khudym @vladislavkeblysh @ruzniaievdm We decided not to unify this component into a `ModalNotification` component, but to reuse it on the Course certificates and Group configuration, Textbook pages.

Reply:
Removed `import React from 'react';`